src/battle/quintohit.php

```diff
@@ -52,8 +54,10 @@
 			$db->execute("update `bixos` set `hp`=`hp`-? where `player_id`=?", [$totalpak, $player->id]);
 		}
 
-		$db->execute("update `players` set `mana`=`mana`-? where `id`=?", [$mana, $player->id]);
+		$player->mana -= $mana; //Correção de bug onde não reduzia a mana em um ataque quadruplo fatal.
+		$db->execute("update `players` set `mana`=? where `id`=?", [$player->mana, $player->id]);
 		array_unshift($_SESSION['battlelog'], "3, Você deu um ataque quádruplo n" . $enemy->prepo . " " . $enemy->username . " e tirou " . $totalpak . " pontos de vida.");
 		$db->execute("update `bixos` set `vez`='e' where `player_id`=?", [$player->id]);
 	}
+	$db->commit();
```

Comment:
This function doesn't exist on AdoDB connection object, we can use commitTrans

https://adodb.org/dokuwiki/doku.php?id=v5:userguide:transactions